projects/kit/components/progress/progress-label/progress-label.style.less

```diff
@@ -0,0 +1,24 @@
+:host {
+    position: relative;
+    display: inline-block;
+    line-height: 0;
+}
+
+::ng-deep progress {
+    width: 100%;
+}
+
+.label {
+    font: var(--tui-font-text-s);
+    color: var(--tui-text-01);
+
+    position: absolute;
```

Comment:
There's a mixin `.fullsize` for those styles

Reply:
fixed